.tooling/markdownlint/config-draft.markdownlint-cli2.cjs

```diff
@@ -9,5 +9,5 @@ module.exports = {
       style: "atx",
     },
   },
-  globs: ["**/*.md", "!node_modules"],
+  globs: ["**/*.md", "!node_modules", "!lo"],
```

Comment:
@bountonw I'm excluding `lo` from linting for now since it has obvious issues.

Reply:
Shouldn't the markdown be enforced? The Thai rules won't apply, but it should check for proper markdown.

Reply:
There's lots of errors like:

```
lo/MB/01_raw/MB02_lo.md:92 MD024/no-duplicate-heading Multiple headings with the same content [Context: "###"]
lo/MB/01_raw/MB02_lo.md:146 MD003/heading-style Heading style [Expected: atx; Actual: atx_closed]
```

We can either exclude those rules or exclude the files.

Reply:
Can we exclude those specific checks that are producing the current errors?